packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+export enum ReadingMode {
+  /**
+   * Do not read the content at all.
+   */
+  DoNotRead,
+
+  /**
+   * The content in this component will read out after other reading tasks of screen reader.
+   * E.g. first read the information of currently focus control, then this content.
+   * NOTE: ReadingMode.ReadAfterOtherContent doesn't support ChromeVOX v53.
+   */
+  ReadAfterOtherContent,
```

Comment:
These enums need numerical assignments. Otherwise changing them in any way can break customers.